src/platform_impl/windows/event_loop.rs

```diff
@@ -324,6 +326,17 @@ fn get_wait_thread_id() -> DWORD {
     }
 }
 
+lazy_static! {
+    static ref WAIT_PERIOD_MIN: UINT = unsafe {
+        let mut tc = TIMECAPS {
+            wPeriodMin: 0,
+            wPeriodMax: 0,
+        };
+        timeapi::timeGetDevCaps(&mut tc as *mut _, mem::size_of::<TIMECAPS>() as _);
```

Comment:
check and handle errors ?
if the call errors out, then wPeriodMin could be undefined.